Increase _bfd_coff_max_nscns to 65279

From: Ray Donnelly <mingw.android@gmail.com>

From
   <https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-binutils/0010-bfd-Increase-_bfd_coff_max_nscns-to-65279.patch>

From 1316395a096fd327efe88c171a08ba077f6f02df Mon Sep 17 00:00:00 2001
Subject: [PATCH 5/5] bfd: Increase _bfd_coff_max_nscns to 65279

.. from 32768. This is the value that llvm uses. There's no
indication in the PECOFF specs that 32768 is the limit.
---
 0 files changed

diff --git a/bfd/coff-alpha.c b/bfd/coff-alpha.c
index 726d20bf..8cbd0c5f 100644
--- a/bfd/coff-alpha.c
+++ b/bfd/coff-alpha.c
@@ -2290,7 +2290,7 @@ static const struct ecoff_backend_data alpha_ecoff_backend_data =
     alpha_ecoff_swap_filehdr_out, alpha_ecoff_swap_aouthdr_out,
     alpha_ecoff_swap_scnhdr_out,
     FILHSZ, AOUTSZ, SCNHSZ, 0, 0, 0, 0, FILNMLEN, TRUE,
-    ECOFF_NO_LONG_SECTION_NAMES, 4, FALSE, 2, 32768,
+    ECOFF_NO_LONG_SECTION_NAMES, 4, FALSE, 2, 65279,
     alpha_ecoff_swap_filehdr_in, alpha_ecoff_swap_aouthdr_in,
     alpha_ecoff_swap_scnhdr_in, NULL,
     alpha_ecoff_bad_format_hook, _bfd_ecoff_set_arch_mach_hook,
diff --git a/bfd/coff-mips.c b/bfd/coff-mips.c
index 3ada9ceb..12a07640 100644
--- a/bfd/coff-mips.c
+++ b/bfd/coff-mips.c
@@ -1312,7 +1312,7 @@ static const struct ecoff_backend_data mips_ecoff_backend_data =
     mips_ecoff_swap_filehdr_out, mips_ecoff_swap_aouthdr_out,
     mips_ecoff_swap_scnhdr_out,
     FILHSZ, AOUTSZ, SCNHSZ, 0, 0, 0, 0, FILNMLEN, TRUE,
-    ECOFF_NO_LONG_SECTION_NAMES, 4, FALSE, 2, 32768,
+    ECOFF_NO_LONG_SECTION_NAMES, 4, FALSE, 2, 65279,
     mips_ecoff_swap_filehdr_in, mips_ecoff_swap_aouthdr_in,
     mips_ecoff_swap_scnhdr_in, NULL,
     mips_ecoff_bad_format_hook, _bfd_ecoff_set_arch_mach_hook,
diff --git a/bfd/coff-rs6000.c b/bfd/coff-rs6000.c
index 85e90a02..25b48e21 100644
--- a/bfd/coff-rs6000.c
+++ b/bfd/coff-rs6000.c
@@ -4093,7 +4093,7 @@ static const struct xcoff_backend_data_rec bfd_xcoff_backend_data =
       3,			/* _bfd_coff_default_section_alignment_power */
       FALSE,			/* _bfd_coff_force_symnames_in_strings */
       2,			/* _bfd_coff_debug_string_prefix_length */
-      32768,			/* _bfd_coff_max_nscns */
+      65279,			/* _bfd_coff_max_nscns */
       coff_swap_filehdr_in,
       coff_swap_aouthdr_in,
       coff_swap_scnhdr_in,
diff --git a/bfd/coff-sh.c b/bfd/coff-sh.c
index 4b6b0de5..e24b0dba 100644
--- a/bfd/coff-sh.c
+++ b/bfd/coff-sh.c
@@ -3101,7 +3101,7 @@ static bfd_coff_backend_data bfd_coff_small_swap_table =
 #else
   2,
 #endif
-  32768,
+  65279,
   coff_swap_filehdr_in, coff_swap_aouthdr_in, coff_swap_scnhdr_in,
   coff_swap_reloc_in, coff_bad_format_hook, coff_set_arch_mach_hook,
   coff_mkobject_hook, styp_to_sec_flags, coff_set_alignment_hook,
diff --git a/bfd/coffcode.h b/bfd/coffcode.h
index c3f13300..824967c3 100644
--- a/bfd/coffcode.h
+++ b/bfd/coffcode.h
@@ -5313,7 +5313,7 @@ static bfd_coff_backend_data bfd_coff_std_swap_table ATTRIBUTE_UNUSED =
 #else
   2,
 #endif
-  32768,
+  65279,
   coff_SWAP_filehdr_in, coff_SWAP_aouthdr_in, coff_SWAP_scnhdr_in,
   coff_SWAP_reloc_in, coff_bad_format_hook, coff_set_arch_mach_hook,
   coff_mkobject_hook, styp_to_sec_flags, coff_set_alignment_hook,
